sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/src/Generated/Models/AccessMode.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ContainerOrchestratorRuntime.Models
+{
+    /// <summary> Storage Class Access Mode. </summary>
+    public readonly partial struct AccessMode : IEquatable<AccessMode>
```

Comment:
Same. Better to add certain prefix, like `StorageClass`.

Reply:
Changed to `StorageClassAccessMode`